src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -64,17 +63,15 @@ public class JsonRpc : IDisposableObservable
         /// </summary>
         private readonly Func<Task, object, JsonRpcMessage> handleInvocationTaskResultDelegate;
 
-        /// <summary>
-        /// A dictionary to map clr method names to <see cref="JsonRpcMethodAttribute" /> values.
-        /// </summary>
-        private readonly ReadOnlyDictionary<string, string> requestMethodToClrMethodMap;
+        private readonly List<Tuple<object, ReadOnlyDictionary<string, string>>> targetRequestMethodToClrMethodMap;
```

Comment:
Can you add/update the xml doc comment for this member to clearly describe the contents of this collection?